library/pk.c

```diff
@@ -416,24 +416,23 @@ int mbedtls_pk_get_psa_attributes(const mbedtls_pk_context *pk,
     }
     more_usage |= PSA_KEY_USAGE_EXPORT | PSA_KEY_USAGE_COPY;
 
+    int want_private = !(usage == PSA_KEY_USAGE_VERIFY_MESSAGE ||
```

Comment:
Given there are 6 legal values for usage, I think it would not be longer to write this with no negation.

Reply:
It would be fewer characters, but harder to read.

Reply:
I'd find it easier to read (that's why I was pointing it out). But since you disagree, it means it's a matter of personal preference, are readability issues sometimes are, so let's leave it at that.